components/function-controller/.gitignore

```diff
@@ -23,3 +23,5 @@ vendor/
 *.swp
 *.swo
 *~
+
+function-controller
```

Comment:
we may think of a dedicated directory for a build files

Reply:
generic makefile defines this directory